udata/commands/fixtures.py

```diff
@@ -30,11 +30,21 @@
 DEFAULT_FIXTURE_FILE = (
     "https://raw.githubusercontent.com/opendatateam/udata-fixtures/main/results.json"  # noqa
 )
+DEFAULT_FIXTURES_RESULTS_FILE = "results.json"
```

Comment:
Could be nice/safer/more readable to use `pathlib.Path` instead of strings filenames for files?

Reply:
Also:

If we keep it as a string and not a Path, I would name it DEFAULT_FIXTURES_RESULTS_FILENAME instead of DEFAULT_FIXTURES_RESULTS_FILE, and I would name results_filename var instead of results_file. Let me know if that makes sense.